### PR TITLE
Add an acknowledgements section to README

### PR DESCRIPTION
**Pull request by erikcs**

### CODE REVIEW

**Comment by swager** [comment]
Thanks! Could you please update the text as follows, and move it above the references:

### Funding
Development of grf is supported by the National Science Foundation, the Sloan Foundation, the Office of Naval Research and Schmidt Futures.

File: README.md [hunk 1]
```diff
@@ -129,3 +129,6 @@ In addition to providing out-of-the-box forests for quantile regression and caus
 Susan Athey, Julie Tibshirani and Stefan Wager.
 <b>Generalized Random Forests</b>, <i>Annals of Statistics</i> 47.2 (2019): 1148-1178.
 [<a href="https://arxiv.org/abs/1610.01271">arxiv</a>]
+
+### Acknowledgements
+Development of grf is made possible by contributions from the National Science Foundation, Sloan Foundation, Office of Naval Research (grant N00014-17- 1-2131) and Schmidt Futures.
```
Comment (jtibshirani) [inline README.md]: Sorry to nitpick, let's use GRF (all caps) to match the rest of the README.

**Comment by susanathey** [comment]
@erikcs Apologies but ONR is very picky about including the grant number.  Can you please adjust to "the Office of Naval Research (Grant N00014-17-1-2131)"

**Comment by erikcs** [comment]
No problem @susanathey it is adjusted in #675

**Pull request merged**